src/pages/questions/index.tsx

```diff
@@ -12,166 +12,224 @@ import { toast } from "react-hot-toast";
 import { useSession } from "next-auth/react";
 
 export default function Questions() {
-    const { data: sessionData } = useSession();
-    const allowedToModify = sessionData?.user.role === "MAINTAINER";
-    const getAllQuery = api.useContext().question.getAll;
-
-    const [viewQns, setViewQns] = useState<QuestionMap>(new Map());
-    const [changedQns, setChangedQns] = useState(new Set<string>());
-    const [deletedQns, setDeletedQns] = useState(new Set<string>());
-
-    const questions = makeMap(api.question.getAll.useQuery(undefined, {
-        onSuccess: (data) => {
-            const mappedData = data.map((q) => ({
-                ...(changedQns.has(q.id) ? viewQns.get(q.id) ?? q : q)
-            }));
-            setViewQns(makeMap(mappedData, 'id'));
-        },
-        onError: (e) => {
-            toast.error("Failed to fetch questions: " + e.message);
-        },
-    }).data ?? [], 'id');
-
-    const hasChanges = (id: string) => {
-        const q1 = questions.get(id);
-        const q2 = viewQns.get(id);
-        return q1 && q2 && !equals(q1, q2);
-    }
-
-    const addMutation = api.question.addOne.useMutation({
-        onError: (e, { title }) => {
-            toast.error(`Failed to add question '${title}':\n\t${e.message}`);
-        }
-    });
-
-    const updateMutation = api.question.updateOne.useMutation({
-        onError: (e, { title }) => {
-            toast.error(`Failed to update question '${title}':\n\t${e.message}`);
-        }
+  const { data: sessionData } = useSession();
```

Comment:
Are these all formatter changes? Can't really tell if there is any change?

Reply:
Can disable format on save?

Reply:
Yeap they are all formatter changes, the error messages for questions is still not yet fixed ><
We need to avoid a runtime error when server is down. 
Since its already formatted here the diffs will be more obvious in the next PR when its been fixed